api/envoy/type/tracing/v3/custom_tag.proto

```diff
@@ -98,5 +98,8 @@ message CustomTag {
 
     // A custom tag to obtain tag value from the metadata.
     Metadata metadata = 5;
+
+    // A response header custom tag.
```

Comment:
nit: please move this directly after `request_header` (before `metadata`).

Reply:
ok， i have fix it